cli/tokio_util.rs

```diff
@@ -1,9 +1,14 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
 pub fn create_basic_runtime() -> tokio::runtime::Runtime {
-  tokio::runtime::Builder::new_current_thread()
+  tokio::runtime::Builder::new_multi_thread()
```

Comment:
This is a huge red flag; Deno's `MainWorker` and `WebWorker` are neither `Sync` nor `Send` (meaning they are pinned to the specific thread) and that's be design (can't move V8 isolate to other thread). I'm surprised that it compiles

Reply:
I was initially as nonplussed as you are but the crux is that we call `block_on()`, that still runs the future on the current thread.

Tasks spawned with `tokio::spawn()` now need to be `Send` (and presumably `Sync`) but deno uses them very sparingly. I don't see anything wrong with this change...

...except that the functions should now be called `create_multi_runtime()` and `run_multi()`, of course. :-)

Aside: there's a copy of this file in `runtime/tokio_util.rs`.